### DISCUSSION TITLE
"ClarityBetter" of the Better Xcloud

### DISCUSSION THREAD

**Discussion by ricardo404** [redphx/better-xcloud#96] [General]
I would like to bring more information about the clarity feature of the Redphx script with some comparisons, see:

Clarity OFF
![0](https://github.com/redphx/better-xcloud/assets/35298519/775c53c6-24a6-47a3-903a-34aff925e8d7)

Clarity 1
![1](https://github.com/redphx/better-xcloud/assets/35298519/6dd6a674-2b81-4b62-ad4c-1d4eacee3eef)

Clarity 2
![2](https://github.com/redphx/better-xcloud/assets/35298519/44f7be21-ce32-4a0c-8598-9ea236f50032)

Clarity3
![3](https://github.com/redphx/better-xcloud/assets/35298519/cbeaf066-a1a1-45ca-986b-edb2f9d90457)

**For better observation, observe the notches of the pyramid.**

In my observations, Clarity ‘2’ and "3" manages to be better than the Clarity of the edge itself. Soon I will put the comparison. Thank you redphx for this incredible feature.

**Comment by redphx** [comment]
I think you should only use 1 or 2, as 3 will destroy the details of the stream.  

Check the Decode time stat, how much does it increase compare to Clarity Boost? But I think Clarity Boost should perform better.

**Reply by ricardo404** [reply]
If it has I haven't noticed. I confess that with your refusal I'm not even using Edge's Clarity anymore. 
I've been testing much of the day and noticed that the Clarity 1, would be equivalent to the Clarity Edge, without a doubt. Using Clarity 2, I'm having a much better visual quality than the edge itself, believe me. 100% sure that it is not placebo effect. 

And another thing, I can now change the saturation, brightness and contrast settings and now I can have spectacular image adjustments, I'll leave it at the attachment. 

A Red suggestion, could add profile options to brightness options. For example, these settings I'm using only look good when taking day flights in the simulator! The night I have to go back to normal settings. Could you create save options for more than one setting of what is set in these settings?

Yes, using option 3 is not a suitable option.

Clarity Booster default Edge:
![Screenshot_34](https://github.com/redphx/better-xcloud/assets/35298519/a1d65067-51ce-4f8c-aef8-7d187e4162ad)

Clarity from Better, option "2" plus contrast and brightness adjustment to leave with "different" look.:
![Screenshot_35](https://github.com/redphx/better-xcloud/assets/35298519/b00acdbf-f524-4357-b9db-d4ea120cedd1)

**Reply by well-santos** [reply]
what contrast and brightness settings do you use?
